### ISSUE TITLE
Need to provide a backup focus indicator for windows high contrast mode

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6155]
### Expected Behavior
People using windows in high contrast mode should be able to see what item has focus.

### Actual Behavior
We are using box-shadow as our focus indicator. The box-shadow property doesn't show up in windows high contrast mode.

More info here:
https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/

### Steps to Reproduce

1. Use windows in high contrast mode
2. Navigate with keyboard and move focus to an element
3. Box-shadow indicator doesn't show up

### Your Environment
- Grommet version: 2.23.0
- Browser Name and version: chrome
- Operating System and version (desktop or mobile): windows

**Comment by jcfilben** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
